flake.nix

```diff
@@ -10,15 +10,11 @@
     inputs.bellroy-nix-foss.lib.haskellProject {
       src = ./.;
       supportedCompilers = [
-        "ghc810"
-        "ghc90"
-        "ghc92"
-        "ghc94"
```

Comment:
I thought our policy was to leave things supported unless it was work to keep them supported? Has something changed here?

Reply:
https://exploring-better-ways.bellroy.com/open-source-at-bellroy-supporting-old-ghc-versions.html